client/my-sites/hosting/php-version-card/index.js

```diff
@@ -0,0 +1,236 @@
+				if ( true === success ) {
+					requestPhpVersion( siteId );
+				}
+				return [ [ requestId( siteId, method ), success ] ];
+			},
+			freshness: 0,
+		}
+	);
+};
+
+const PhpVersionCard = ( {
+	translate,
+	siteId,
+	loading,
+	updating,
+	updateResult,
+	version,
+	showErrorNotice,
+	showSuccessNotice,
+} ) => {
+	const [ selectedPhpVersion, setSelectedPhpVersion ] = useState( null );
+
+	useEffect( () => {
+		requestPhpVersion( siteId );
+	}, [] );
+
+	useEffect( () => {
+		const updateNoticeId = 'hosting-php-version';
+
+		if ( updateResult === 'failure' ) {
+			showErrorNotice( translate( 'Failed to set PHP version.' ), {
+				id: updateNoticeId,
+			} );
+		}
+
+		if ( updateResult === 'success' ) {
+			showSuccessNotice(
+				translate( 'PHP version successfully set to %(version)s.', {
+					args: {
+						version: selectedPhpVersion,
+					},
+				} ),
+				{
+					id: updateNoticeId,
+					showDismiss: false,
+					duration: 5000,
+				}
+			);
+		}
+	}, [ updateResult ] );
+
+	const changePhpVersion = event => {
+		const newVersion = event.target.value;
+
+		setSelectedPhpVersion( newVersion );
+	};
+
+	const getPhpVersions = () => {
+		return [
+			{
+				label: translate( '7.2', {
+					comment: 'PHP Version',
+				} ),
+				value: '7.2',
+			},
+			{
+				label: translate( '7.3 (recommended)', {
+					comment: 'PHP Version',
+				} ),
+				value: '7.3',
+			},
+		];
+	};
+
+	const getContent = () => {
+		if ( loading ) {
+			return;
+		}
+
+		const isButtonDisabled = ! selectedPhpVersion || selectedPhpVersion === version;
+		let buttonTooltip = undefined;
+		if ( isButtonDisabled ) {
+			buttonTooltip = translate( "You're already running PHP %(version)s.", {
+				args: {
+					version,
+				},
+			} );
+		}
+
+		return (
+			<div>
+				<p>{ translate( 'Manage PHP versions.' ) }</p>
+
+				<Accordion
+					title={ translate( 'How it affects your site?' ) }
+					className="php-version-card__accordion"
+				>
+					<p>{ translate( 'In all kinds of weird ways.' ) }</p>
+				</Accordion>
+
+				<div>
+					<FormLabel>{ translate( 'Your site is currently running:' ) }</FormLabel>
+					<FormSelect
+						className="php-version-card__version-select"
+						onChange={ changePhpVersion }
+						defaultValue={ version }
+					>
+						{ getPhpVersions().map( option => {
+							return (
+								<option
+									disabled={ option.value === version }
+									value={ option.value }
+									key={ option.label }
+								>
+									{ option.label }
+								</option>
+							);
+						} ) }
+					</FormSelect>
+				</div>
+				<Button
+					className="php-version-card__set-version"
+					onClick={ () => setPhpVersion( siteId, selectedPhpVersion ) }
+					busy={ updating }
+					disabled={ isButtonDisabled }
+					title={ buttonTooltip }
+				>
+					<span>{ translate( 'Update PHP Version' ) }</span>
+				</Button>
+			</div>
+		);
+	};
+
+	return (
+		<Card className="php-version-card">
+			<MaterialIcon icon="dns" size={ 32 } />
```

Comment:
Can we use a different icon here? This one is already used by the database card.

@olaolusoga is there any Material icon that could work here? https://material.io/resources/icons/?style=baseline

Reply:
Using `build`.